src/main/resources/migrations.xml

```diff
@@ -647,9 +647,31 @@
 	</changeSet>
 
 	<changeSet id="35" author="evdhoudt" dbms="mssql" >
-		<sql dbms="mssql">
+		<sql dbms="">
 			INSERT INTO adminSettings ([key], value) VALUES ('EXTERNAL_DOCUMENTATION_LINK_URL', '');
 		</sql>
 	</changeSet>
 
+	<changeSet  id="36" author="evdhoudt">
```

Comment:
Give your changeset a meaningful name instead of a sequential number (see the last three migrations on candidate).